### PR TITLE
Hollow/apples&broken sfx fix

### PR DESCRIPTION
**Pull request by abbclas**

### CODE REVIEW

File: app/Assets/Other/Nilkuza Trial.otf
Comment (noh0st) [inline app/Assets/Other/Nilkuza Trial.otf:1]: whats this

Comment (noh0st) [reply app/Assets/Other/Nilkuza Trial.otf:1]: oh a font

File: app/Scripts/AppleSpawner.gd
Comment (noh0st) [inline app/Scripts/AppleSpawner.gd:1]: probably want the apples to spawn in a range around the player, can change it here or merge this and make a separate card for that if we think we want that.

**Review by noh0st — APPROVED** [review]

File: app/Scripts/AppleSpawner.gd
Comment (abbclas) [reply app/Scripts/AppleSpawner.gd:1]: alr i'll merge first

**Pull request merged**